app/controllers/users_controller.rb

```diff
@@ -1,12 +1,15 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  before_action :load_user, only: %i(show edit update destroy)
+  before_action :logged_in_user, only: %i(edit update destroy)
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i(edit update destroy)
  before_action :load_user, only: %i(show edit update destroy)
```
chỗ này mình nên check xem user đã login chưa trc khi load tài nguyên mới là hợp lý nhất nhé